src/main/java/Parser.java

```diff
@@ -0,0 +1,70 @@
+import java.util.Scanner;
+
+public class Parser {
+    static Scanner sc = new Scanner(System.in);
+
+    /**
+     * Reads the next line of input.
+     * @return the String that has been read
+     */
+    public static String readLine() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Checks if there is still input in the next line
+     * @return true if there is, false if not
+     */
+    public static boolean stillHasInputs() {
+        return sc.hasNext();
+    }
+
+    /**
+     * Extracts and returns the command from the input line
+     * @param input A string of a single command, or a command with arguments
+     * @return the command represented as a String
+     */
+    public static String getCommand(String input) {
+        String[] inputs = input.split(" ", 2);
+        return inputs[0];
+    }
+
+    /**
+     * Extracts and returns the arguments from the input line
+     * @param input A String of command with arguments
+     * @return the arguments represented as a String
+     */
+    public static String getArgs(String input) {
+        String[] inputs = input.split(" ", 2);
+        return inputs[1];
+    }
```

Comment:
not too sure, but i think this might throw an IndexOutOfBoundsException if your input is a one word command